src/mono/wasm/Wasm.Build.Tests/Blazor/BlazorWasmProjectProvider.cs

```diff
@@ -21,39 +13,18 @@ public BlazorWasmProjectProvider(ITestOutputHelper _testOutput, string? _project
             : base(_testOutput, _projectDir)
     {}
 
-    public void AssertBlazorBootJson(
-        string config,
-        bool isPublish,
-        string targetFramework = BuildTestBase.DefaultTargetFrameworkForBlazor,
-        bool expectFingerprintOnDotnetJs = false,
-        RuntimeVariant runtimeType = RuntimeVariant.SingleThreaded)
-    {
-        AssertBootJson(binFrameworkDir: FindBlazorBinFrameworkDir(config, isPublish, targetFramework),
-                      isPublish: isPublish,
-                      expectFingerprintOnDotnetJs: expectFingerprintOnDotnetJs,
-                      runtimeType: runtimeType);
-    }
-
-    public void AssertBlazorBundle(
-        BlazorBuildOptions options,
-        bool isPublish,
-        string? binFrameworkDir = null)
-    {
-        EnsureProjectDirIsSet();
-        if (options.TargetFramework is null)
-            options = options with { TargetFramework = BuildTestBase.DefaultTargetFrameworkForBlazor };
-
-        AssertDotNetNativeFiles(options.ExpectedFileType,
-                                      options.Config,
-                                      forPublish: isPublish,
-                                      targetFramework: options.TargetFramework,
-                                      expectFingerprintOnDotnetJs: options.ExpectFingerprintOnDotnetJs,
-                                      runtimeType: options.RuntimeType);
-
-        AssertBlazorBootJson(config: options.Config,
-                             isPublish: isPublish,
-                             targetFramework: options.TargetFramework,
-                             expectFingerprintOnDotnetJs: options.ExpectFingerprintOnDotnetJs,
-                             runtimeType: options.RuntimeType);
-    }
+    public void AssertBundle(BlazorBuildOptions options)
+        => AssertBundle(new AssertWasmSdkBundleOptions(
+                Config: options.Config,
+                IsPublish: options.IsPublish,
+                TargetFramework: options.TargetFramework,
+                BinFrameworkDir: FindBinFrameworkDir(options.Config, options.IsPublish, options.TargetFramework),
+                GlobalizationMode: GlobalizationMode.Default,
+                PredefinedIcudt: null,
+                ExpectFingerprintOnDotnetJs: options.ExpectFingerprintOnDotnetJs,
+                ExpectedFileType: options.ExpectedFileType,
+                RuntimeType: options.RuntimeType,
+                AssertIcuAssets: false, // FIXME: this is broken right now
```

Comment:
That's the place where we have an issue with too many ICU files?

Reply:
yeah